layer5-ng/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@Tanuj22 we might be able to be rid of this file, I suppose. What do you think?

Reply:
Yup. Will remove in the next PR